docs/tutorials/developer-journey/level-3/3.3-certified-data.md

```diff
@@ -0,0 +1,351 @@
+# 3.3: Certified data
+
+## Overview
+
+In a previous module, [2.2: Advanced canister calls](../level-2/2.2-advanced-canister-calls.md), we briefly touched on certified variables. In this guide, we'll dive deeper into certified data and take a look at an example that displays how to use certified data. 
+
+Recall that certified variables are used as a way for queries to return an authenticated response that can be validated and trusted, since query calls do not go through consensus and therefore their response cannot be certified. 
```

Comment:
It may be worthwhile to touch upon the value proposition of certified variables without going into "selling" territory. why should developers care about them?